src/prng/fortuna.lisp

```diff
@@ -75,52 +72,16 @@
                                          event))
     (incf (slot-value pool 'length) (length event))))
 
-(defmethod internal-write-seed (path (pseudo-random-number-generator
```

Comment:
This was moved into `prng.lisp`, since it's actually common to any seedable PRNG.